dhall-csv/dhall-csv.cabal

```
@@ -0,0 +1,63 @@
+Name: dhall-csv
+Version: 0.0.1
+Cabal-Version: >=1.10
+Build-Type: Simple
+Tested-With: GHC == 8.2.2, GHC == 8.4.3, GHC == 8.6.1
+License: BSD3
+License-File: LICENSE
+Copyright: 2021 Marcos Lerones
+Author: Marcos Lerones
+Maintainer: Gabriel439@gmail.com
+Bug-Reports: https://github.com/dhall-lang/dhall-haskell/issues
+Synopsis: Convert bidirectionally between Dhall and CSV files.
+Description:
+    Use this package if you want to convert between Dhall expressions and CSV. 
+    You can use this package as a library or an executable:
+    .
+    * See the "Dhall.CSV" or "Dhall.CSVToDhall" modules if you want to use
+      this package as a library
+    .
+    * Use the @dhall-to-csv@ or @csv-to-dhall@ programs from
+      this package if you want an executable
+    .
+    The "Dhall.CSV" and "Dhall.CSVToDhall" modules also contains instructions
+    for how to use this package
+Category: Compiler
+Extra-Source-Files:
+    CHANGELOG.md
+Source-Repository head
+    Type: git
+    Location: https://github.com/dhall-lang/dhall-haskell/tree/master/dhall-csv
+
+Library
+    Hs-Source-Dirs: src
+    Build-Depends:
+        base                      >= 4.8.0.0  && < 5
```

I'd suggest that we try to support GHC 8.6 and up, since 8.6 is what we use in CI. I don't expect that we'll get requests to support any older GHC versions.

https://gitlab.haskell.org/ghc/ghc/-/wikis/commentary/libraries/version-history is a good resource for the core package versions.

```suggestion
        base                      >= 4.12.0.0  && < 5
```